pagerduty/resource_pagerduty_event_orchestration_path_unrouted.go

```diff
@@ -75,14 +75,14 @@ func resourcePagerDutyEventOrchestrationPathUnrouted() *schema.Resource {
 													Optional:     true,
 													ValidateFunc: validateEventOrchestrationPathEventAction(),
 												},
-												"variables": {
+												"variable": {
 													Type:     schema.TypeList,
 													Optional: true,
 													Elem: &schema.Resource{
 														Schema: eventOrchestrationPathVariablesSchema,
 													},
 												},
-												"extractions": {
+												"extraction": {
```

Comment:
I have checked ruleset example https://github.com/PagerDuty/terraform-provider-pagerduty/blob/master/pagerduty/resource_pagerduty_ruleset_rule.go#L246 and it looks like extractions kept as plural there. Same with `subconditions`. Is that overlooked in rulesets?

Reply:
I believe it was either overlooked or added before the naming convention was established because variables that were added later are singular: https://github.com/PagerDuty/terraform-provider-pagerduty/blob/master/pagerduty/resource_pagerduty_ruleset_rule.go#L285